inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -110,33 +138,144 @@ <h1>
                       </button>
                     </div>
                   </form>
-                </div>
+                </span>
               </span>
               <span class="dropdown" aria-haspopup="true" aria-expanded="false">
-                <button class="btn btn-action btn-secondary dropdown-toggle flex-content" type="button" data-toggle="dropdown">
-                  <i class="fas fa-cog"></i>&nbsp;<wicket:message key="settings" />
+                <button class="btn btn-action btn-secondary dropdown-toggle flex-content dd-toggle"
+                  type="button">
+                  <i class="fas fa-user"></i>&nbsp;
+                  <wicket:message key="Annotators"/>
                 </button>
-                <div class="dropdown-menu shadow-lg pt-0 pb-0" role="menu" style="min-width: 800px;">
-                  <div class="card-header">
-                    <wicket:message key="settings" />
+                  <script>
+                      let dropDownMenuButton = $('button.dd-toggle');
+
+                      dropDownMenuButton.on('click', function () {
+                          $('form.ddform').show();
+                      });
+
+                      $('body').on('click', function (e) {
+                          if (!dropDownMenuButton.is(e.target)
+                              && !$('form.ddform').is(e.target)
+                              && !$('li').is(e.target)
+                              && dropDownMenuButton.has(e.target).length === 0
+                              && $('form.ddform').has(e.target).length === 0
+                              && $('.open').has(e.target).length === 0) {
+                              $('form.ddform').hide();
+                          }
+                      });
+                  </script>
+                <form wicket:id="userForm" class="shadow-lg pt-0 pb-0 ddform" role="menu"
+                  style="width: 1000px; height: 435px;">
```

Comment:
Hard-coding the with to 1000 might not be a good idea, e.g. for people that have small screens. Can we use a relative width?

Reply:
I tried a lot. If it gets even with some adjustments below 800px, the complete layout would not look good (missing words in the dropdown etc.)
Is 800px fine?
And as it is only a dropdown, there is no need to grow if there is more space, because especially when we have a lot of space, it might be good to see more from the background (the table)

Reply:
We can go with 800px for the moment and if necessary see if/which other options there may be in a future UI update. Thanks!.